tools/bin/acceptance_test_kube_helm.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+
+set -e
+
+. tools/lib/lib.sh
+
+assert_root
+
+# Since KIND does not have access to the local docker agent, manually load the minimum images required for the Kubernetes Acceptance Tests.
+# See https://kind.sigs.k8s.io/docs/user/quick-start/#loading-an-image-into-your-cluster.
+if [ -n "$CI" ]; then
+  echo "Loading images into KIND..."
+  kind load docker-image airbyte/server:dev --name chart-testing &
+  kind load docker-image airbyte/webapp:dev --name chart-testing &
+  kind load docker-image airbyte/worker:dev --name chart-testing &
+  kind load docker-image airbyte/db:dev --name chart-testing &
+  kind load docker-image airbyte/container-orchestrator:dev --name chart-testing &
+  kind load docker-image airbyte/bootloader:dev --name chart-testing &
+  wait
+fi
+
+echo "Replacing default Chart.yaml with a test one"
+mv charts/airbyte/Chart.yaml charts/airbyte/Chart.yaml.old
+mv charts/airbyte/Chart.yaml.test charts/airbyte/Chart.yaml 
```

Comment:
Why do we need to use a test one?

Reply:
To support local testing + acceptance

Reply:
I did that on purpose, before testing make the .test chart.yaml main one and after reset everything to normal.
So we can run this script both locally and on CI server

Reply:
got it. how is someone supposed to apply this then?

Reply:
and how does this setting this to .test affect publishing?